extensions-core/druid-ranger-security/pom.xml

```diff
@@ -211,6 +457,47 @@
                     </usedDependencies>
                 </configuration>
             </plugin>
+            <plugin>
```

Comment:
Remove the build plugin, it was needed to build the extension separately from Druid, it will be built as part of Druid now